Utilities/bootstrap

```diff
@@ -198,6 +201,11 @@ class Target(object):
                                              "usr/lib/swift/CoreFoundation"))
             import_paths.append(os.path.join(args.foundation_path,
                                              "usr/lib/swift"))
+        if args.libdispatch_build_dir:
+            import_paths.append(os.path.join(args.libdispatch_build_dir,
+                                             "src"))
```

Comment:
I haven't looked at the Linux libdispatch build at all yet, but what imports is this satisfying? Is it C header imports?